src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,8 @@
+//colors
+$mainColor: #060b35;
+$secondaryColor: #616070;
+$hoverColor: #00acdc;
```

Comment:
Better use abstract naming. Because maybe, you will need this color as a background next time
That's why use sth like "accentColor"